File: 2025/problems/eieio/submissions/accepted/atli.cpp
#include<iostream>
using namespace std;

int main() {
    int a, b;
    cin >> a >> b;
    int res = b - 2 * a;
    if(res < 0 || res % 2 != 0 || (a - res / 2 < 0))
        cout << "Rong talning\n";
    else cout << res / 2 << '\n';
}

File: 2025/problems/eieio/submissions/accepted/atli.py
#!/usr/bin/python3
a = int(input())
b = int(input())
res = b - 2 * a
if res < 0 or res % 2 != 0 or (a - res // 2) < 0:
    print("Rong talning")
else:
    print(res // 2)
